crates/runtime/tests/s3/mod.rs

```diff
@@ -126,6 +126,60 @@ async fn s3_federation() -> Result<(), anyhow::Error> {
         .await
 }
 
+#[tokio::test]
+async fn s3_pdfs() -> Result<(), anyhow::Error> {
```

Comment:
Looks like you will need to merge from trunk? I thought this code was already merged.